### ISSUE TITLE
Solar Tweaks launching regular lunar

### ISSUE THREAD

**Issue by MegaGuyGG** [Solar-Tweaks/Solar-Tweaks#174]
My Solar Tweaks launching a regular lunar.
![Снимок экрана (17)](https://user-images.githubusercontent.com/105493541/168271186-688de1c2-f98e-4e52-9438-5ef7ee12ff13.png)
![Снимок экрана (18)](https://user-images.githubusercontent.com/105493541/168271192-43d49f84-5a86-45a6-9e0f-e9d8a05ec9c6.png)
![Снимок экрана (19)](https://user-images.githubusercontent.com/105493541/168271197-529cd86c-b7b1-45c5-9f78-c67dbd187ab8.png)
![Снимок экрана (20)](https://user-images.githubusercontent.com/105493541/168271198-601a0159-3c0d-4af4-9443-e6b33aeea98e.png)
![Снимок экрана (21)](https://user-images.githubusercontent.com/105493541/168271203-267f7ef7-e5a6-4e3d-a821-a71e62d79eec.png)

**Comment by Khonshulol** [comment]
Yes we understand but the solar api is down rn so the functionality will be limited.

**Comment by davidlol01** [comment]
when are you guys expecting it to be back?

**Comment by 770grappenmaker** [comment]
I dunno, our server didnt get renewed, which also handles updates
